src/TrimDeps/Program.cs

```diff
@@ -0,0 +1,58 @@
+using System.IO;
+using System.Linq;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+public class Program
+{
+    public static void Main(string[] args)
+    {
+        ChangeEntryPointLibraryName(args[0], null);
```

Comment:
I'm a little confused what is trying to be achieved here. From the code below it looks like you are only removing the target library which is the first entry under targets/tfm and libraries. In that case why bother with the second parameter? Also is it always guaranteed that the first entry is what we want to trim? Is it better to be logging for the specific package  *HostingStartup.Template I presume?